wherescape/logging.py

```diff
@@ -108,7 +110,13 @@ def initialise_wherescape_logging(wherescape):
     w_handler.setFormatter(message_format)
     logger.addHandler(w_handler)
 
-    f_handler = logging.FileHandler(f"{wherescape.workdir}wherescape.log")
+    # File log rotation will occur Saturday night 23:59
+    f_handler = TimedRotatingFileHandler(
+        f"{wherescape.workdir}python_logging\wherescape.log",
+        when="W5",
+        atTime=time(23, 59),
```

Comment:
Any reason why you've not chosen `midnight` ( ref:  [TimedRotatingFileHandler](https://docs.python.org/3/library/logging.handlers.html#rotatingfilehandler:~:text=as%20described%20previously.-,TimedRotatingFileHandler,-%C2%B6)  )?